helm/install/templates/manager-upgrade.yaml

```
@@ -16,6 +16,9 @@ spec:
     metadata:
       labels:
         {{- include "install.upgradeLabels" . | nindent 8 }}
+        {{- if .Values.customLabels }}
```

indent of toYaml should be same then around (for easier read) and just write once `.Values.customLabels`

```suggestion
        {{- with .Values.customLabels }}
        {{- toYaml . | indent 8 }}
        {{- end }}
```